hr_payroll_extended/models/hr_payroll_extended.py

```diff
@@ -209,6 +206,4 @@ def get_worked_day_lines(
     def create(self, cr, uid, vals, context=None):
```

Comment:
I think this *create* method should be removed since it's almost only doing *super()*